_docs/overview.md

```diff
@@ -2,59 +2,55 @@
-     Webhooks | |  REST API      Webhooks | |  REST API
-              V |                         V |
-   +-------------------+          +-------------------+
-   | modelservice sim1 |          | modelservice sim2 |
-   +-------------------+          +-------------------+
-        A           A                 A           A
-        | WAMP      | WAMP            | WAMP      | WAMP
-        V           V                 V           V
-+-----------+  +-----------+  +-----------+  +-----------+
-| Browser 1 |  | Browser 2 |  | Browser 3 |  | Browser 4 |
-+-----------+  +-----------+  +-----------+  +-----------+
-     A               A             A               A
-     | Static        | Static      | Static        | Static
-     | assets        | assets      | assets        | assets
-   +-------------------+         +-------------------+
-   |   frontend sim1   |         |   frontend sim2   |
-   +-------------------+         +-------------------+
+The browser will use the frontend service as a presentation layer, to authenticate users, and to connect to the model service in order to send and receive events through a WebSocket using [WAMP Protocol](http://wamp-proto.org/).
 
+The model service will listen for events from the browser and, as needed, make REST API calls through the simpl-client to the `simpl-games-api` service (e.g. to store state).
 
-```
+Every time data is created, modified, or deleted in the `simpl-games-api` service, it will send an event to the registered game model service via HTTP Webhook.
 
-## The WAMP Layer
+The model service will automatically forward the event to the browser and the state will update.
 
-The communication between the modelservice and the browser happens via Websocket by using the [WAMP Protocol](http://wamp-proto.org/).
+`simpl-games-api` currently supports basic and session authentication. 
+You will need to create one or more users for your modelservice(s) to use connecting to the `simpl-games-api`. 
 
-Thw WAMP Protocol is mainly composed of two patterns: publish/subscribe (**PubSub** for short) and call/register (or **RPC**).
+### Simpl Defined
 
-### PubSub vs RPC
+Roles
+* Leader: oversees the game
+* Player: plays the game
+
+Single-player Versus Multi-player
+* Single-player game: each player can see only his or her own decisions and results. A run’s leaders can see the decisions and results of all the run’s players.
+* Multi-player game: all players belonging to the same world can see the decisions and results of that world. A run’s leaders can see the decisions and results of all the run’s worlds.
+
+Game Terminology
+
+<!--
+* Run:
+* Scenario:
+-->
+* Period: happens within a scenario
+* Decision: a choice a player makes
+* Result: the effect created by one or more decisions
+* `Play` phase: players can submit decisions
+* `Debrief` phase: players are prevented from submitting further decisions. The leader can review the final decisions and results of all worlds in the run.
+
```

Comment:
Play and Debrief are not built in. They just happen to be the way Learning Lab has structured its game phases.